validation-test/stdlib/DictionaryTrapsObjC.swift

```diff
@@ -205,6 +198,10 @@ DictionaryTraps.test("ForcedNonverbatimBridge.Value")
   _ = d1 as! Dictionary<NSObject, String>
 }
 
+// FIXME: <rdar://problem/49026133>
+// Segmentation fault: 11 (when compiling with optimizations).
+// While running pass #0 SILModuleTransform "SILGenCleanup".
```

Comment:
Hm frustratingly, this compiler crash is a new issue that is unrelated to the original problem (which was a runtime abort() call in the `*_debug.sanity` tests). Serves us right for leaving this test disabled for so long... 😞

If you still have a crash log handy, could you please file a bug for it?

Reply:
> If you still have a crash log handy, could you please file a bug for it?

I've filed SR-14282.